packages/grub2/build.yaml

```diff
@@ -28,8 +27,7 @@ steps:
 
 {{if eq .Values.name "grub2-efi-image"}}
 - mkdir -p /EFI/BOOT
-- cp config/grub_efi.cfg.tmpl /EFI/BOOT/grub.cfg.tmpl
-- cos-tmpl-render /EFI/BOOT/grub.cfg.tmpl
+- cp config/grub_efi.cfg /EFI/BOOT/grub.cfg
```

Comment:
If this file is not copied here then the `grub2-mkimage` call should also not make any reference to it: `-c /EFI/BOOT/grub.cfg` parameter can be dropped.

Reply:
Probably drop whole `grub2-efi-image`